README.md

```diff
@@ -65,14 +83,30 @@ Once you have installed and set up the requirements, there are three steps to de
     python -m pip install -r requirements.txt
     ```
 
-    This will install Ansible and other Python dependencies. Then, from the same directory, type:
+    This will install Ansible and other Python dependencies. You can do this within a conda environment which you can create and activate with:
+
+    ```bash
+    conda create -n xnat-aws-9 -c conda-forge python=3.9
+    conda activate xnat-aws-9
+    ```
```

Comment:
Not a strong feeling, but do we really need this? There's no need to use `conda` and the purpose of this doc is not teach someone how to use `conda`